tests/docker/globus.ubuntu24.Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:24.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=true
+
+#### Get and install iRODS repo ####
+RUN apt-get update && apt-get install -y wget gnupg2 lsb-release && \
+    wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add - && \
+    echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list && \
+    apt-get update
+
+#### Install icommands - used to set up, validate and tear down tests. ####
+#### Install externals and dev package to build the connector.         ####
+
+# Comment out the irods-icommands and irods-dev if testing against locally
+# built packages.
+RUN apt-get install -y \
+    irods-icommands \
+    irods-dev \
+    irods-externals-cmake3.21.4-\
+    irods-externals-clang13.0.1-0 \ 
+    irods-externals-clang-runtime13.0.1-0
+
+#### install basic packages ####
+RUN apt-get update && apt-get install -y curl \
+    g++ \
+    gcc \
+    ftp \
+    telnet \
+    libcurl4-openssl-dev \
+    unzip \
+    python3 \
+    python3-distro \
+    python3-psutil \
+    python3-jsonschema \
+    python3-requests \
+    python3-pip \
+    python3-pyodbc
+
+
+#### Get and install globus repo ####
+RUN wget -q https://downloads.globus.org/globus-connect-server/stable/installers/repo/deb/globus-repo_latest_all.deb && \
+    dpkg -i globus-repo_latest_all.deb && \
+    apt-get update
```

Comment:
Use `apt-get` instead of `dpkg` to install the package. This way, any unmet dependencies will be pulled in. You will have to pass the full path to the file. This applies to any other similar usage of `dpkg`.

Additionally, delete the package file after it has been installed. This applies to any other case where a package is downloaded in a Dockerfile RUN block.

Reply:
done